CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## How to contribute to guides.rubygems.org
+
+#### **If you want to contribute to RubyGems itself, visit [the RubyGems Guides](http://guides.rubygems.org/contributing/)**
+This guide just covers the details of contributing to the guides themselves.
+
+### Running guides.rubygems.org
```

Comment:
How about `Setting up a local development environment` instead? I find that more clearer.